cli/enry/main.go

```diff
@@ -197,10 +278,30 @@ func printFileAnalysis(file string) {
 	)
 }
 
-func getLines(file string, content string) (int, int) {
-	totalLines := strings.Count(content, "\n")
-	nonBlank := totalLines - strings.Count(content, "\n\n")
-	return totalLines, nonBlank
+func getLines(content []byte) (total int, nonBlank int) {
```

Comment:
It will be a bit nicer to use `bufio.Scanner` here because it will not require loading the whole file into memory and the code for blank line detection would be simpler:
```golang
line := scanner.Bytes()
line = bytes.TrimSpace(line)
isBlank := len(line) == 0
```